### PR TITLE
Add missing metrics back in

### PR DESCRIPTION
**Pull request by paladique**
The "Pull Requests Opened" Metric is missing from the details view. Adding it back in. Also cleared the help docs metrics counter, as it seems that they weren't being cleared.

### CODE REVIEW

File: src/GitHub.VisualStudio/Services/UsageTracker.cs [hunk 1]
```diff
@@ -353,6 +353,9 @@ static void ClearCounters(UsageModel usage, bool weekly, bool monthly)
             usage.NumberOfForkPullRequestsCheckedOut = 0;
             usage.NumberOfForkPullRequestPulls = 0;
             usage.NumberOfForkPullRequestPushes = 0;
+            usage.NumberOfGitHubPaneHelpClicks = 0;
+            usage.NumberOfWelcomeTrainingClicks = 0;
+            usage.NumberOfWelcomeDocsClicks = 0;
```
Comment (grokys) [inline src/GitHub.VisualStudio/Services/UsageTracker.cs]: Oh! How embarrassing! This means these metrics up until now are completely wrong...

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs [hunk 2]
```diff
@@ -362,6 +361,8 @@ public override void Initialize([AllowNull] ViewWithData data)
             }
 
             IsLoading = IsBusy = false;
+            IsBusy = false;
```
Comment (grokys) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: This line just repeats what the line above is already doing.

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs [hunk 3]
```diff
@@ -362,6 +361,8 @@ public override void Initialize([AllowNull] ViewWithData data)
...
+            usageTracker.IncrementPullRequestOpened().Forget();
```
Comment (grokys) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: This method actually gets called when the pane is refreshed too, we probably don't want to increment the counter there. I think you can tell if it's a new pull request when `Model == null` on entry to the method. So I'd set a flag there and only increment if the flag is set.

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs (hunk 2 above)
Comment (paladique) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: ahhh a cherry pick/merge mistake!

File: src/GitHub.VisualStudio/Services/UsageTracker.cs (hunk 1 above)
Comment (marlazinger200) [inline src/GitHub.VisualStudio/Services/UsageTracker.cs]: 1

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs (hunk 2 above)
Comment (marlazinger200) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: 1

**Comment by marlazinger200** [comment]
1

File: src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs (hunk 3 above)
Comment (marlazinger200) [inline src/GitHub.App/ViewModels/PullRequestDetailViewModel.cs]: 1

**Pull request merged**